.github/dependabot.yml

```diff
@@ -0,0 +1,116 @@
+version: 2
+
+updates:
+  - directory: /
+    package-ecosystem: cargo
+    schedule:
+      interval: daily
+    rebase-strategy: auto
+
+  - directory: /
+    package-ecosystem: docker
```

Comment:
I have no clue if this will catch build-tools.Dockerfile and compute-node.Dockerfile